src/azul/deployment.py

```diff
@@ -91,18 +91,15 @@ def es_endpoint(self) -> Optional[Netloc]:
 
     @property
     def es_instance_count(self) -> Optional[int]:
-        if config.es_endpoint:
-            return config.es_instance_count
+        return config.es_instance_count
 
     @property
     def lambda_env(self) -> Mapping[str, str]:
         if config.share_es_domain:
-            endpoint = "${data.aws_elasticsearch_domain.azul_es_domain.endpoint}"
-            instance_count = "${data.aws_elasticsearch_domain.azul_es_domain.cluster_config[0].instance_count}"
+            instance_count = "${var.azul_es_domain.cluster_config[0].instance_count}"
         else:
-            endpoint = "${var.azul_es_domain.endpoint}"
             instance_count = config.es_instance_count
```

Comment:
If the comment above is true then this can just be removed. 

Otherwise... I'm wondering why this is using `config` directly instead of using the property here: https://github.com/DataBiosphere/azul/pull/1856/commits/edf1c22e8f1f10841d4003281704c6f0ea6c4022#diff-747ee2f562a8dc659d6f60c2b431abbeR93. Do you know why the property above exists / can it be removed?